iohk-monitoring/src/Cardano/BM/Configuration.lhs

```diff
@@ -13,6 +13,8 @@ module Cardano.BM.Configuration
     , CM.minSeverity
     , CM.setMinSeverity
     , CM.inspectSeverity
+    , CM.getGitCommit
```

Comment:
why not a generic function "setOption :: Configuration -> Text -> Text -> IO ()" ?
can be used to set the commit hash and the appversion.

Reply:
I considered it, but having seen the bunch of disparate functions I guessed the lack of genericity as intended.

Reply:
Done.